bridge/opentracing/bridge_test.go

```diff
@@ -546,3 +546,34 @@ func TestBridge_SpanContext_IsSampled(t *testing.T) {
 		})
 	}
 }
+
+func TestOTHTTPHeadersCarrier(t *testing.T) {
+	carrier := ot.HTTPHeadersCarrier{}
+	testBridgeWithCarrier(t, carrier)
+}
+
+func TestOTTextMapCarrier(t *testing.T) {
+	carrier := ot.TextMapCarrier{}
+	testBridgeWithCarrier(t, carrier)
+}
+
+func testBridgeWithCarrier(t *testing.T, carrier interface{}) {
+	mockOtelTracer := internal.NewMockTracer()
+	bridgeTracer, _ := NewTracerPair(mockOtelTracer)
+	otel.SetTextMapPropagator(propagation.TraceContext{})
+	span := bridgeTracer.StartSpan("testSpan1")
+	// adding data to baggage
+	span.SetBaggageItem("foo", "bar")
+	defer span.Finish()
+
+	err := bridgeTracer.Inject(span.Context(), ot.HTTPHeaders, carrier)
+	assert.Nil(t, err, "Inject error")
```

Comment:
assert.Nil is not wrong but this is generally used for checking whether any var holds nil value, that's why it has a third argument for description. For error check, there is a dedicated function provided by the library which is `asssert.NoError(t, err)`. The same is being used throughout the unit tests already (refer line 87, 125, 137).

Reply:
That make sense.
Just one small question, is it fine if i provide third argument in `asssert.NoError(t, err, msg)` ? 
Something like this
assert.NoError(t, err, "Inject error")

Reply:
Yes, that's a good idea. The relevant message might it easier to figure out where the error happened, in case test fails.

Reply:
done